style.css

```diff
@@ -52,6 +52,27 @@ h1, h2 {
+			vertical-align: -3px;
+			margin-right: 3px;
+			opacity: .5;
+		}
+
+		.mdn-link::before {
+			background: url('data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0iVVRGLTgiPz4KPHN2ZyB3aWR0aD0iMTYiIGhlaWdodD0iMTYiIHZlcnNpb249IjEuMSIgdmlld0JveD0iMCAwIDQuMjMzMyA0LjIzMzMiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgeG1sbnM6Y2M9Imh0dHA6Ly9jcmVhdGl2ZWNvbW1vbnMub3JnL25zIyIgIHhtbG5zOnJkZj0iaHR0cDovL3d3dy53My5vcmcvMTk5OS8wMi8yMi1yZGYtc3ludGF4LW5zIyI+CjxwYXRoIGQ9Im0yLjQwNzYgMC42NzA3Yy0wLjUxMDQxLTAuMjk3MjQtMS4wODMzLTAuMTQ4NTYtMS40NDQ5IDAuMTY3MDRsLTAuOTYwNSAwLjk0MzggMC42NTE0Ny0wLjAxNjcwNC0wLjM2NzUgMC40MjU5NiAwLjY2ODE3IDAuMDE2NzA0LTAuNDE3NjEgMC40NDI2NyAwLjU2Nzk1IDAuMDA4MzUyMS0wLjE1MDM0IDAuNDc2MDdjMC40MTg0MSAwLjQxMjgzIDAuOTA3NTQgMC42NDg4MyAxLjQxOTkgMC44MjY4NiAwLjE0MzctMC41NDk1OS0wLjA0Njk1NC0xLjM4MTggMC40NTkzNy0xLjQzNjYgMC4zMzM0OS0wLjAzNjEwNyAwLjU1MDE0IDAuMTI3MDcgMC43NTE3IDAuMDkxODc0czAuMjU4OTItMC4xNjcwNCAwLjI1ODkyLTAuMTY3MDRjMC4yMDUyMSAwLjA1MDYyIDAuMjU2MTUtMC4xNTU4NyAwLjMzNDA5LTAuMzE3MzggMC4wMTc5NzgtMC4xMjM3NyAwLjAwMTE2MDYtMC4yMTI3NS0wLjAyNTA1Ni0wLjI5MjMzIDAuMDA4NTUyNS0wLjE2NDc2IDAuMDIxNTItMC4zMzE1MS0wLjIyMDExLTAuMzg5MDktMC4xNjI3NS0wLjA2MzUzMS0wLjMwNzg3LTAuMTMzOTgtMC41MTQ4OC0wLjIxMjI2LTAuMTE3NjMtMC4wMzY0MTYtMC4yMjI5NS0wLjEzNTYyLTAuMjgzOTctMC4zNTkxNC0wLjAzNjcwNy0wLjEzNDQ2LTAuMTQyMDctMC4yMDgwMi0wLjI5MzY3LTAuMjE4MDgtMC4wODkwMDgtMC4wMDU5MDYtMC4yODE3OCAwLjA5NzMyMy0wLjQzMjk3IDAuMDA5Mjc4eiIgZmlsbD0iI2ZlZmVmZSIgZmlsbC1ydWxlPSJldmVub2RkIi8+Cjwvc3ZnPgo=') 0 0 / 16px 16px;
+		}
+		
+		.whatwg-link::before {
+			background: url('data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0iVVRGLTgiPz4KPHN2ZyB4bWxuczpjYz0iaHR0cDovL2NyZWF0aXZlY29tbW9ucy5vcmcvbnMjIiB4bWxuczpkYz0iaHR0cDovL3B1cmwub3JnL2RjL2VsZW1lbnRzLzEuMS8iIHhtbG5zOnJkZj0iaHR0cDovL3d3dy53My5vcmcvMTk5OS8wMi8yMi1yZGYtc3ludGF4LW5zIyIgdmVyc2lvbj0iMS4xIiB2aWV3Qm94PSIwIDAgMTAwIDEwMCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KPGNpcmNsZSBjeD0iNTAiIGN5PSI1MCIgcj0iNDUiIGZpbGw9Im5vbmUiIHN0cm9rZT0iI2ZmZiIgc3Ryb2tlLXdpZHRoPSIxMCIvPgo8cGF0aCBkPSJtMzggMzhjMC0xMiAyNC0xNSAyMy0yIDAgOS0xNiAxMy0xNiAyM3Y3aDEwdi00YzAtOSAxNy0xMiAxNy0yNy0yLTIyLTQ1LTIyLTQ1IDN6bTcgMzJoMTB2MTBoLTEwIiBmaWxsPSIjZmZmIi8+CjxzY3JpcHQvPgo8L3N2Zz4K') 0 0 / 16px 16px;
```

Comment:
As I mentioned earlier, part of the reason to use SVG (besides scalability), is that they are readable directly in the CSS. If you base64 encode them, this defeats that benefit. They do not need base64 encoding! You can just paste them directly in a data URI, and only a few special characters need to be URL-encoded (like `#` to `%23`). 
For an example, see: https://github.com/LeaVerou/stretchy/blob/60dd3384123ffc15d4311a9f3fef1da808402f55/style.css#L113

Reply:
Oh, I see. :)